django/contrib/admin/options.py

```diff
@@ -56,6 +56,12 @@
 HORIZONTAL, VERTICAL = 1, 2
 
 
+class AdminTemplateResponse(TemplateResponse):
+    def __init__(self, request, template, context=None, content_type=global_settings.DEFAULT_CONTENT_TYPE, status=None,
+                 charset=None, using=None):
```

Comment:
Assuming we move forward with this solution, it might be better to use `*args, **kwargs` and `kwargs.setdefault('content_type', '...')` so we don't have  update the signature here if `TemplateResponse` changes.